requirements.txt

```diff
@@ -1 +1,2 @@
-pep8==1.5.7
\ No newline at end of file
+pep8==1.5.7
+testfixtures==4.1.2
```

Comment:
I saw that you commented the test with the testfixtures, so this is not being used :)